bpf/tests/bpf_tests/Makefile

```diff
@@ -0,0 +1,49 @@
+# Copyright Authors of Cilium
```

Comment:
Yes, you are right. Will fix that. I did it like this because when I started we had other tests in the `bpf/tests` dir with would mess with what I was doing. Not needed anymore since I have replaced all existing tests.